### PR TITLE
feat(core): `TextfieldController` add prefix, postfix and filler

### LINKED ISSUE #3067: 🚀 -  Add prefix/postfix to TextfieldController
### Which @taiga-ui/* package(s) are relevant/releated to the feature request?

kit

### Description

I want to be able to use prefix/postfix properties of PrimitiveTexfield in more complex controls to be able to do something like this:
`<tui-input prefix='https://'></tui-input>`

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #3067 
## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/demo/src/modules/components/input-count/input-count.template.html [hunk 1]
```diff
@@ -133,18 +133,24 @@
                 documentationPropertyName="prefix"
                 documentationPropertyMode="input"
                 documentationPropertyType="string"
+                [documentationPropertyDeprecated]="true"
                 [(documentationPropertyValue)]="prefix"
             >
-                A prefix symbol, like currency
+                A prefix symbol, like currency Use
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/input-count/input-count.template.html:139]:
```suggestion
                A prefix symbol, like currency. Use
```

Comment (nsbarsukov) [reply projects/demo/src/modules/components/input-count/input-count.template.html:139]: (and other places too)

File: projects/demo/src/modules/components/input-slider/input-slider.template.html [hunk 2]
```diff
@@ -238,10 +241,31 @@
                 documentationPropertyName="postfix"
                 documentationPropertyMode="input"
                 documentationPropertyType="string"
+                [documentationPropertyDeprecated]="true"
                 [(documentationPropertyValue)]="postfix"
             >
                 A postfix symbol, like currency
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/input-slider/input-slider.template.html:247]:
```
Use tuiTextfieldPostfix instead
```

File: projects/demo/src/modules/components/input-slider/input-slider.template.html [hunk 3]
```diff
@@ -229,6 +231,7 @@
                 documentationPropertyName="prefix"
                 documentationPropertyMode="input"
                 documentationPropertyType="string"
+                [documentationPropertyDeprecated]="true"
                 [(documentationPropertyValue)]="prefix"
             >
                 A prefix symbol, like currency
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/input-slider/input-slider.template.html:237]:
```
Use tuiTextfieldPrefix instead
```

File: projects/kit/components/input-slider/input-slider.component.ts [hunk 4]
```diff
@@ -92,24 +96,36 @@ export class TuiInputSliderComponent
...
     @tuiDefaultProp()
-    prefix = ``;
+    textfieldPrefix = ``;
 
-    @Input()
+    /** @deprecated use `tuiTextfieldPostfix` from {@link TuiTextfieldControllerModule} instead */
+    @Input(`postfix`)
     @tuiDefaultProp()
-    postfix = ``;
+    texfieldPostfix = ``;
```
Comment (nsbarsukov) [inline projects/kit/components/input-slider/input-slider.component.ts:107]:
```suggestion
    textfieldPostfix = ``;
```

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo/src/modules/components/input-number/input-number.template.html [hunk 5]
```diff
@@ -259,6 +268,26 @@ <h3>There are also other components to input numbers:</h3>
             >
                 Show/hide decimal
             </ng-template>
+            <ng-template
+                i18n
+                documentationPropertyName="tuiTextfieldPrefix"
```
Comment (waterplea) [inline projects/demo/src/modules/components/input-number/input-number.template.html:273]: Shouldn't we add those new props to the section with all other textfield controller props?

**Review by waterplea — APPROVED** [review]

File: projects/demo/src/modules/components/input-number/input-number.template.html (hunk 5 above)
Comment (vladimirpotekhin) [reply projects/demo/src/modules/components/input-number/input-number.template.html:273]: will be moved to TextfieldControllerDocumentation after #3241

**Pull request merged**